actions/python-black-lint/action.yml

```
@@ -0,0 +1,17 @@
+# Define action name, description, and inputs
+name: "Python Black Lint"
+description: "Lint python code with black"
+
+# Define action steps
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.x"
+        architecture: x64
+
+    - run: pip install black
+      shell: bash
+    - run: black --check .
+      shell: bash
```

I don't think you need to defined shell specifically here

```suggestion
    - run: pip install black
    - run: black --check .
```

---

You do need to specific it with composite actions or you will get an error.